packages/pds-ember/app/styles/pds/components/loading/index.scss

```diff
@@ -36,6 +36,7 @@
 
     svg path {
       stroke-width: 9%; // FIXME: magic number
+      fill: none;
```

Comment:
Was this in the structure-icons styles?

Reply:
No, this was a result of Icon styling applying...

```css
.pds-icon svg { fill: currentColor; }
```